src/sentry/services/hybrid_cloud/organization/impl.py

```diff
@@ -393,7 +393,9 @@ def get_or_create_default_team(
         if team_query.exists():
             team = team_query[0]
         else:
-            team = Team.objects.create(organization_id=organization_id, slug=new_team_slug)
```

Comment:
This would have created a team with no name.

Reply:
i thought name was deprecated?

Reply:
It is deprecated but still exists on the model

Reply:
do we still need to be setting it?

Reply:
for team slug it doesn't really matter b/c we don't use it or surface it on the UI, but it doesn't hurt setting it to the slug. In our api responses, name will match slug which makes more sense than it being empty

Reply:
Nothing broke because of this but I set it because it was ugly 😛